exampleSite/content/docs/faq.md

```diff
@@ -12,7 +12,7 @@ menu:
     name: FAQ
 ---
 
```

Comment:
One more thing: trim single spaces at the end of lines. See [285#pullrequestreview-893651388](https://github.com/Vimux/Mainroad/pull/285#pullrequestreview-893651388) for more info.

Reply:
Will do!